.github/pull_request_template.md

```diff
@@ -1,20 +1,22 @@
 <!--
 If you are a first-time contributor to MontePy,
-refer the developing guidelines at:
+refer to the developing guidelines at:
 https://idaholab.github.io/MontePy/developing.html
 -->
 
 # Description
 
-Please include a summary of the change and which issue is fixed if applicable. Please also include relevant motivation and context.
+Please include a summary of the change and which issue is fixed, if applicable. Please also include relevant motivation and context.
 
 Fixes # (issue)
 
 # Checklist
 
-- [ ] I have performed a self-review of my own code
-- [ ] I have made corresponding changes to the documentation (if applicable)
-- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
+- [ ] I have performed a self-review of my own code.
+- [ ] The code follows the standards outlined in the [development documentation](https://idaholab.github.io/MontePy/developing.html).
+- [ ] I have made corresponding changes to the documentation, providing clear details on the added or modified functionality (if applicable).
```

Comment:
I was thinking this would have some sub bullet points:

``` suggested
    - [ ] I have documented all added Classes and methods
    - [ ] For infrastructure updates I have updated the developer's guide
    - [ ] For significant new features I have added a section to the getting started guide. 
```